pycypher/README.md

```diff
@@ -0,0 +1,46 @@
+# pycypher
+[![Build Status](https://travis-ci.org/cleishm/libcypher-parser.svg?branch=master)](https://travis-ci.org/cleishm/libcypher-parser)
+[![Version](https://img.shields.io/pypi/v/pycypher.svg)](https://pypi.python.org/pypi/pycypher)
+
+Python bindings for libcypher-parser.
+
+### Getting started
+You don't need to have libcypher-parser installed to use it.
+The distribution contains a statically linked copy of libcypher-parser. You
```

Comment:
Needs updating if it's integrated with the source tree?

Reply:
What do you mean?

Reply:
Ok, I think I understand what you mean. The word "distribution" referes here to Python wheels which are generated by build. This is not stored in the source tree.

I think static linking is important so that users will not have to worry about compatiblity between version of the library and the bindings, and will not have to install the library system-wide to use it from Python.

Reply:
The source distribution contains only the code, not a statically linked copy of the library. If this file is only relevant in the binary distribution, it might be good to change the name in the source tree so that the casual reader isn't assuming it's about the source.

Reply:
(Wrote that comment before seeing your last one. Seems fine to me, with the suggestion to rename the file remaining.)

Reply:
Ok, I will change the wording here so this will be more clear. Right now there are no source distributions of the bindings on PyPI and I think this will not change.

Reply:
Great. You could also just rename the file, so that it doesn't look like a readme intended for whomever downloads the source tree.

Reply:
Well, a Readme is not only for developers downloading the source tree, right?
For example, here: https://github.com/cleishm/libcypher-parser/blob/master/README.md
there are instructions for end users on how to install from Homebrew or PPA and how to use it.
So I did something similar, treating the Readme as a homepage of the project. I made PyPI page link to this readme (well, the link will become valid after merging this PR).

Reply:
Ok, sounds fine. Just adjust the language then, so that it doesn't sound like there should be a statically linked library bundled with the source (e.g. "The PyCypher packages include a statically linked version of libcypher-parser, so there is no need to install it separately").

Reply:
Done, I changed the wording.